sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/RecognizeLinkedEntityAsyncClient.java

```diff
@@ -194,15 +202,22 @@
      */
     private Mono<TextAnalyticsPagedResponse<RecognizeLinkedEntitiesResult>> getRecognizedLinkedEntitiesResponseInPage(
         Iterable<TextDocumentInput> documents, TextAnalyticsRequestOptions options, Context context) {
-        return service.entitiesLinkingWithRestResponseAsync(
+        return service.entitiesLinkingWithResponseAsync(
             new MultiLanguageBatchInput().setDocuments(Transforms.toMultiLanguageInput(documents)),
+            context.addData(AZ_TRACING_NAMESPACE_KEY, COGNITIVE_TRACING_NAMESPACE_VALUE),
             options == null ? null : options.getModelVersion(),
-            options == null ? null : options.isIncludeStatistics(),
-            context.addData(AZ_TRACING_NAMESPACE_KEY, COGNITIVE_TRACING_NAMESPACE_VALUE))
+            options == null ? null : options.isIncludeStatistics())
             .doOnSubscribe(ignoredValue -> logger.info("A batch of documents - {}", documents.toString()))
             .doOnSuccess(response -> logger.info("Recognized linked entities for a batch of documents - {}",
                 response.getValue()))
             .doOnError(error -> logger.warning("Failed to recognize linked entities - {}", error))
-            .map(this::toTextAnalyticsPagedResponse);
+            .map(this::toTextAnalyticsPagedResponse)
```

Comment:
Just to confirm I'm understanding this: For single document calls, you're now throwing on errors?

Reply:
Currently, with new swagger changes, the service returns `TextAnalyticsErrrorException`, Which will not explore to user. We want to explore `HttpResponseException` to user for the exception return from service side. In async APIs, we never throw exception but return Mono/Flux errors. 

For your question: " For single document calls, you're now throwing on errors?"
If you are talking about, "if there is partial success in a batch operation, throwing an error", the answer is noop. 
.onErrorMap() will only be triggeed when the service restful call `entitiesLinkingWithResponseAsync` has failed, which is the request call failure.

Reply:
If the error happen that early, can you map the error in early phase before mapping page result?

Reply:
this is the earliest stage. service.entitiesLinkingWithResponseAsync() is the restful API, when it throws exceptions. we map the exception directily.